homeassistant/components/aurora_abb_powerone/sensor.py

```diff
@@ -0,0 +1,98 @@
+"""Support for Aurora ABB PowerOne Solar Photvoltaic (PV) inverter."""
+
+import logging
+import voluptuous as vol
+
+from aurorapy.client import AuroraSerialClient, AuroraError
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_ADDRESS, CONF_NAME, DEVICE_CLASS_POWER, POWER_WATT)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_COMPORT = 'rs485'
+DEFAULT_ADDRESS = 2
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_COMPORT): cv.string,
+    vol.Optional(CONF_ADDRESS, default=DEFAULT_ADDRESS): cv.positive_int,
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
```

Comment:
Please rename `add_devices` to `add_entities`.

Reply:
Ok, but also custom_components/example_sensor/sensor.py should be updated in that case.

Reply:
Updated in e72db09a9